### PR TITLE
Prepare release 1.4.2.

### PR DESCRIPTION
**Pull request by DimCitus**
See #580

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+### pg_auto_failover v1.4.2 (February 3, 2021) ###
+
+This is a bugfix release for v1.4 series
+
+#### Added
+* Implement pg_autoctl do azure commands (QA tooling). [#544]
+* pg autoctl show settings [#549]
+* Improve docker images (build, release) [#556]
```
Comment (hanefi) [inline CHANGELOG.md:8]: nit, only some of the items end with a dot at the end. This is not so easy on my eyes 👀

**Review by hanefi — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (DimCitus) [reply CHANGELOG.md]: Fixed.

**Review by hanefi — CHANGES_REQUESTED** [review]
The vendored in libraries seem to break builds for PG10 and PG11. I think we need to fix them before proceeding.

```
gcc -std=c99 -D_GNU_SOURCE -g -I /usr/include/postgresql -I /usr/include/postgresql/10/server -I /usr/include/postgresql/internal -Wall -Wmissing-prototypes -Wpointer-arith -Wdeclaration-after-statement -Wendif-labels -Wmissing-format-attribute -Wformat-security -fno-strict-aliasing -fwrapv -fexcess-precision=standard -Wno-format-truncation -g -g -O2 -fstack-protector-strong -Wformat -Werror=format-security -fno-omit-frame-pointer -Wformat -Wall -Werror=implicit-int -Werror=implicit-function-declaration -Werror=return-type -Wno-declaration-after-statement -Wno-missing-braces -I/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg -I/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/log/src/ -I/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/subcommands.c/ -I/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/libs/ -I/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/parson/  -c -MMD -MP -MF/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl//.deps/lib-strerror.Po -MTlib-strerror.o -o lib-strerror.o /home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:35:1: warning: no previous prototype for ‘pg_strerror’ [-Wmissing-prototypes]
 pg_strerror(int errnum)
 ^~~~~~~~~~~
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c: In function ‘pg_strerror’:
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:37:27: error: ‘PG_STRERROR_R_BUFLEN’ undeclared (first use in this function)
  static char errorstr_buf[PG_STRERROR_R_BUFLEN];
                           ^~~~~~~~~~~~~~~~~~~~
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:37:27: note: each undeclared identifier is reported only once for each function it appears in
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:39:9: error: implicit declaration of function ‘pg_strerror_r’; did you mean ‘pg_strerror’? [-Werror=implicit-function-declaration]
  return pg_strerror_r(errnum, errorstr_buf, sizeof(errorstr_buf));
         ^~~~~~~~~~~~~
         pg_strerror
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:37:14: warning: unused variable ‘errorstr_buf’ [-Wunused-variable]
  static char errorstr_buf[PG_STRERROR_R_BUFLEN];
              ^~~~~~~~~~~~
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c: At top level:
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:46:1: warning: no previous prototype for ‘pg_strerror_r’ [-Wmissing-prototypes]
 pg_strerror_r(int errnum, char *buf, size_t buflen)
 ^~~~~~~~~~~~~
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:46:1: error: conflicting types for ‘pg_strerror_r’
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:39:9: note: previous implicit declaration of ‘pg_strerror_r’ was here
  return pg_strerror_r(errnum, errorstr_buf, sizeof(errorstr_buf));
         ^~~~~~~~~~~~~
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c: In function ‘pg_strerror’:
/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl/../lib/pg/strerror.c:40:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: some warnings being treated as errors
Makefile:76: recipe for target 'lib-strerror.o' failed
make[2]: *** [lib-strerror.o] Error 1
make[2]: Leaving directory '/home/travis/build/citusdata/pg_auto_failover/src/bin/pg_autoctl'
Makefile:7: recipe for target 'pg_autoctl' failed
make[1]: *** [pg_autoctl] Error 2
make[1]: Leaving directory '/home/travis/build/citusdata/pg_auto_failover/src/bin'
Makefile:81: recipe for target 'bin' failed
make: *** [bin] Error 2
```

**Review by hanefi — APPROVED** [review]
Everything looks ok now. I think we can merge this one after the tests pass.

**Pull request merged**